android_common/src/main/res/values/strings.xml

```diff
@@ -93,7 +93,8 @@ as the name makes more sense. If there is a local specific name, make sure to wr
   <string name="Czech">Czech</string>
   <string name="Slovak">Slovak</string>
   <string name="Russian">Russian</string>
-  <string name="Latvian">Latvian</string>
+  <string name="Latvian">Latvian Traditional</string>
+  <string name="LatvianExt">Latvian Extended</string>
```

Comment:
The key value should be `Latvian_Extended`. Everything that goes into the `strings.xml` file has the same key as the value so it is easier for the translators to pick up :)